### PR TITLE
PotPlayer

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**

### CODE REVIEW

File: pending/xnview.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-19
+
+-->
+<cleaner id="xnview.new">
```
[the remark this answers is not in the document]
Comment (Tobias-B-Besemer) [reply pending/xnview.xml]: Fixed.

File: pending/potplayer.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-20
+
+-->
+<cleaner id="potplayer">
```
[the remark this answers is not in the document]
Comment (Tobias-B-Besemer) [reply pending/potplayer.xml]: Done.

File: pending/potplayer.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  <option id="rememberfiles">
+    <label>Remember Files</label>
+    <description>Delete the Remember Files list</description>
+        <action command="winreg" path="HKCU\Software\Daum\PotPlayerMini\RememberFiles\*" />
+        <action command="winreg" path="HKCU\Software\Daum\PotPlayerMini64\RememberFiles\*" />
+  </option>
+  <option id="logs">
+    <label>Log Files</label>
+    <description>Delete the Log Files</description>
+        <action command="delete" search="file" path="$APPDATA\Daum\PotPlayer\Log\*"/>
```
Comment (az0) [inline pending/potplayer.xml]:
```suggestion
        <action command="delete" search="glob" path="$APPDATA\Daum\PotPlayer\Log\*"/>
```

File: pending/potplayer.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  <option id="mru">
+    <label>Most recently used</label>
+    <description>Delete the list of recently used documents</description>
+    <action command="winreg" path="HKCU\Software\Daum\PotPlayerMini\RememberFiles\"/>
+    <action command="winreg" path="HKCU\Software\Daum\PotPlayerMini64\RememberFiles\"/>
+  </option>
+  <option id="logs">
+    <label>Logs</label>
+    <description>Delete the logs</description>
+    <action command="delete" search="walk.files" path="%AppData%\Daum\PotPlayer\Log\*"/>
```
Comment (az0) [inline pending/potplayer.xml]: Do you mean every directory under `%AppData%\Daum\PotPlayer\Log\`? If yes, then `walk.files` is right, but if you want every file then you want either

 `search="glob"` (which will not delete recursively)
OR
remove the wildcard  (which will delete recursively)

File: pending/potplayer.xml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+	v0.1 done on 2018-12-20
+
+-->
+<cleaner id="potplayer" os="windows">
+  <label>PotPlayer</label>
+  <description>Media Player</description>
+  <option id="rememberfiles">
+    <label>Remember Files</label>
+    <description>Delete the Remember Files list</description>
+        <action command="winreg" path="HKCU\Software\Daum\PotPlayerMini\RememberFiles\*" />
```
[the remark this answers is not in the document]
Comment (Tobias-B-Besemer) [reply pending/potplayer.xml]: OK.

File: pending/potplayer.xml (hunk 3 above)
Comment (Tobias-B-Besemer) [reply pending/potplayer.xml]: What's about `search="walk.files"`?

File: pending/potplayer.xml (hunk 4 above)
Comment (Tobias-B-Besemer) [reply pending/potplayer.xml]: OK, fixed it!

File: pending/potplayer.xml (hunk 3 above)
Comment (az0) [reply pending/potplayer.xml]: If you want to recursively delete all files under the directory `Log`, yes: use walk.files and take off the wildcard.

Comment (Tobias-B-Besemer) [reply pending/potplayer.xml]: Yes, did that!

**Pull request merged**